rmw_connextdds_common/include/rmw_connextdds/rmw_impl.hpp

```diff
@@ -597,6 +602,9 @@ class RMW_Connext_Client
     DDS_Publisher * const pub,
     DDS_Subscriber * const sub,
     const rosidl_service_type_support_t * const type_supports,
+#ifndef TRACETOOLS_DISABLED
+    const rmw_client_t * const rmw_client,
+#endif  // TRACETOOLS_DISABLED
```

Comment:
this change is causing uncrustify to require this other change:

```diff
--- include/rmw_connextdds/rmw_impl.hpp
+++ include/rmw_connextdds/rmw_impl.hpp.uncrustify
@@ -599 +599 @@
-  create(
+    create(
```

same with the similar change below. Very weird.

Reply:
I've just decided to remove the `#ifndef`/`#endif`